java/ql/src/semmle/code/java/frameworks/play/Play.qll

```diff
@@ -0,0 +1,167 @@
+import java
+
+/**
+ * Play MVC Framework Result Class
```

Comment:
Qldoc for classes needs to start with a noun-phrase. This means that in most cases it should start with either `A ...` or `The ...`, depending on what's appropriate. Also, qldoc should generally form sentences that are properly terminated with a `.`.